e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -60,7 +60,7 @@ Scenario('Gatekeeper enters allocation decision', async (I, caseViewPage, enterA
   I.seeEventSubmissionConfirmation(config.applicationActions.enterAllocationDecision);
 });
 
-Scenario('Gatekeeper enters hearing details and submits', async (I, caseViewPage, loginPage, addHearingBookingDetailsEventPage) => {
```

Comment:
Do we need to do this test here? We do the same test in `e2e/tests/hmctsAdministersCaseAfterSubmission_test.js:161`. Should one of them be removed?